Sources/MapboxDirections/RouteOptions.swift

```diff
@@ -429,6 +435,11 @@ open class RouteOptions: DirectionsOptions {
             params.append(URLQueryItem(name: CodingKeys.waypointTargets.stringValue, value: targetCoordinates))
         }
         
+        if waypoints.first(where: { $0.layer != nil} ) != nil {
```

Comment:
Nit: `contains(where:_:)` is adequate since we aren’t using the return value for anything.